EasyPost/Billing.cs

```diff
@@ -27,9 +27,9 @@ public static async Task<bool> DeletePaymentMethod(PaymentMethod.Priority priori
         /// <param name="amount">Amount to fund.</param>
         /// <param name="priority">Which type of payment method to use to fund the wallet. Defaults to primary.</param>
         /// <returns>True if successful, false otherwise.</returns>
-        public static async Task<bool> FundWallet(string amount, PaymentMethod.Priority priority = PaymentMethod.Priority.Primary)
+        public static async Task<bool> FundWallet(string amount, PaymentMethod.Priority? priority)
```

Comment:
I'd prefer we expose the default to the user and not hide it in the function. Can't it be a nullable default? If not then I say (if we can), we just leave the default here.

Reply:
You can't use a non-static variable as a default value in a method declaration (one downside to using a custom Enum class rather than the built-in one)

Reply:
Actually, I was mistaken about the reason.
You can't have default values of anything other than `null` and strings in C#:
<img width="949" alt="image" src="https://user-images.githubusercontent.com/17054780/191130511-114d6786-41ae-43d5-bdf5-b96f6a6092c5.png">

Reply:
The docstring for this method indicates that "primary" is the default value if one is not provided.

Reply:
That's super annoying, ok.